### PR TITLE
KevS-SK1326-Create-Manage-DNS-history-for-DomainDNS-TRANSLATIONS

### PR DESCRIPTION
**Pull request by Kazarus4**
Draft for translations.

DO NOT MERGE BEFORE https://github.com/ovh/docs/pull/5494.

### CODE REVIEW

**Comment by Kazarus4** [comment]
translations are done => need proof for : 

- DE and EN by @tcpdumpfbacke 
- Latin and PL by @Y0Coss 

DO NOT MERGE BEFORE https://github.com/ovh/docs/pull/5494. I'll merge the two PRs when the new feature will be available in manager (probably at the begining of the next week).

**Review by bs-ovh — APPROVED** [review]

File: pages/web_cloud/domains/dns_zone_history/guide.pl-pl.md
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_history/guide.pl-pl.md]:
```suggestion
updated: 2023-12-11
```

File: pages/web_cloud/domains/dns_zone_history/guide.pl-pl.md
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_history/guide.pl-pl.md]:
```suggestion
- Dostęp do [panelu klienta OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.pl/&ovhSubsidiary=pl)
```

**Pull request merged**
